packages/fxa-auth-server/lib/routes/password.js

```diff
@@ -481,7 +489,11 @@ module.exports = function (
           request.emitMetricsEvent('password.forgot.send_code.start'),
           customs.check(request, email, 'passwordForgotSendCode'),
         ])
+
           .then(db.accountRecord.bind(db, email))
+          .then((account) => {
+            return account
+          })
```

Comment:
Q: What dis? Looks like a noop since we're just returning the previously resolved promise chain value (unless I'm missing something obvious).

Reply:
I was debugging something, it is not necessary...